vendor/settingsClient.js

```diff
@@ -0,0 +1,305 @@
+/* eslint-disable */
+import axios from 'axios'
```

Comment:
if this is a vendor thing with no npm page, where does this come from ? how to update it ?

Reply:
added a comment